src/metabase/models/recent_views.clj

```diff
@@ -0,0 +1,87 @@
+(ns metabase.models.recent-views
```

Comment:
This file has the most amount of new business logic in the PR, so it's the most important area to focus on for review